UniversalAlgebra/Algebras/Setoid.lagda

```diff
@@ -78,5 +84,49 @@ record SetoidAlgebra α ρ : Type (𝓞 ⊔ 𝓥 ⊔ lsuc (α ⊔ ρ)) where
      --       1. a function  f : (⟦ 𝑆 ⟧s Den) .Carrier  → Den . Carrier
      --       2. a proof cong : f Preserves _≈₁_ ⟶ _≈₂_ (that f preserves the setoid equalities)
 
+
+\end{code}
+
+#### Products of SetoidAlgebras
+
+\begin{code}
+
+module _ {α ρ ι : Level} where
+
+ open SetoidAlgebra
+
+ ⨅ : {I : Type ι }(𝒜 : I → SetoidAlgebra α ρ) → SetoidAlgebra _ _ -- (𝓘 ⊔ α) 𝑆
+ open IsEquivalence renaming ( refl  to  reflE
+                             ; sym   to  symE
+                             ; trans to  transE )
+
+ Den (⨅ {I} 𝒜) .∣_∣ = ∀ i → ∣ Den (𝒜 i) ∣
```

Comment:
For this, I would have used the `record` syntax instead of the copattern version. For this, with the right `open`s, it is much nicer.